<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tohum Sayma</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCameraOpen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamerayı Aç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCameraClose">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Kamerayı Kapat</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYeni">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionA"/>
    <addaction name="actionKaydet"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_lemler">
    <property name="title">
     <string>İşlemler</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSe_enekler">
    <property name="title">
     <string>Seçenekler</string>
    </property>
   </widget>
   <addaction name="menuYeni"/>
   <addaction name="menu_lemler"/>
   <addaction name="menuSe_enekler"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionA">
   <property name="text">
    <string>Aç</string>
   </property>
  </action>
  <action name="actionKaydet">
   <property name="text">
    <string>Kaydet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
